web/models/releases.ex

```diff
@@ -1,5 +1,6 @@
 defmodule HexWeb.Releases do
   use HexWeb.Web, :crud
+  alias HexWeb.{Assets, Sitemaps, RegistryBuilder}
```

Comment:
I was thinking about adding crud module aliases only to controller macro and not the shared macro - so that cruds are not aliased in the schemas and views etc where they prob shouldn't be ever used. so I'd leave this unchanged and just make the change for controllers. wdyt? I don't have a strong opinion about it tho